scripts/refresh-abis.sh

```
@@ -79,32 +79,65 @@ cleanup() {
-  local out="$4"
   local wt
   wt="$(mktemp -d -t "$(basename "$repo")-${tag}-XXXXXX")"
-  WORKTREES+=("$repo::$wt")
-  echo "  Worktree: $repo @ $tag -> $wt"
+  echo "  Worktree: $repo @ $tag -> $wt" >&2
   git -C "$repo" worktree add --detach "$wt" "$tag" >/dev/null
   # Submodules (lib/solady, lib/v4-core, etc.) aren't materialized by `worktree add`.
   # Rewrite git@github.com URLs to https so we don't need an SSH key configured here,
   # then init submodules in-place so forge can resolve remappings.
   git -C "$wt" \
     -c url."https://github.com/".insteadOf="git@github.com:" \
```

Shellcheck (0.11.0)

[warning] Word is of the form "A"B"C" (B indicated). Did you mean "ABC" or "A\"B\"C"?

(SC2140)